### DISCUSSION TITLE
Preseted filter disabled time values

### DISCUSSION THREAD

**Discussion by Clems92** [AdguardTeam/AdGuardHome#4450] [Ideas]
Hi all!

I switched from another ads blocking solution to AdGuardHome, so happy to use it right now!

Nevertheless, I'm missing some intersting fonctiunalities, the most useful one is the ability to easily disable the filter for a preseted period of time, using the web interface. 
Instead of having the "disable" button on the dashboard, we could have a rolling menu after clicking on "disable" with values like "disable for 5 / 30 / 60 / Cutom". 

Sometimes it is useful to disable the filter for a few time to acces to a content, and quite often I forget to reactivate it!

Thanks!

**Comment by ainar-g** [comment]
Sorry about the late response.  This sounds like feature request #2823.

**Reply by Clems92** [reply]
Hello ainar-g, 

thanks for your aswer. True it is a bit of https://github.com/AdguardTeam/AdGuardHome/issues/2823 

I give more details here https://github.com/AdguardTeam/AdGuardHome/issues/4451
